### ISSUE TITLE
Buchungsliste nur partiell aktualisieren

### ISSUE THREAD

**Issue by tolot27** [openjverein/jverein#528]
Momentan wird die Buchungsliste nach einer Zuordnung von Buchungsart, Sollbuchung, Projekt bzw. Kontoauszug komplett aktualisiert und man muss wieder an die Stelle scrollen, wo man vorher war. Das ist recht mühsam.
Meiner Ansicht nach ist es ausreichend, wenn nur die Zeilen der geänderten Buchung(en) aktualisiert werden.
Lässt ich das irgendwie umsetzen? Vielleicht gibt es das ja bereits bei anderen Views?

**Comment by tolot27** [comment]
Diese Funktionalität gibt es bereits. Wenn man nämlich aus einer Splittbuchung zurückgeht (nachdem man sie erstellt hat), hat die originale Buchung in der Liste den Fokus und die neuen Splittbuchungen werden auch aufgeführt.

**Comment by lenilsas** [comment]
Ja, das ist möglich, ich habe mich schon mal damit beschäftigt bin aber noch nicht dazu gekommen es umzusetzen da ich noch mit anderen Änderungen beschäftigt bin.
